datanode/gateway/graphql/schema.graphql

```diff
@@ -4161,6 +4161,8 @@ input OrderFilter {
   "Date range to retrieve orders from/to. Start and end time should be expressed as an integer value of nano-seconds past the Unix epoch"
   dateRange: DateRange
   excludeLiquidity: Boolean
+  "If true, only orders that are live will be returned."
```

Comment:
Would be good to know exactly what 'live' means. Is it just STATUS_ACTIVE, STATUS_PARKED and STATUS_PARTIALLY_FILLED?